src/main.ts

```diff
@@ -52,6 +51,9 @@ const server = httpApi.listen(config.port, () => {
 })
 
 console.log('PFI DID: ', config.did.id)
+fs.writeFileSync('server-did.txt', config.did.id)
+
+
```

Comment:
i believe a new DID is already created at the bottom of `config.ts` if one wasn't provided through environment variables. should we just adjust things there to write DID and associated key material to disk?

Reply:
yep - take a look again